src/main/java/Duke.java

```diff
@@ -1,10 +1,171 @@
+    public static final String SPECIFY_TIME_WARNING = "Please specify time!";
+    public static final String ILLEGAL_CMD_WARNING = "No such command! Please try again!";
+    public static final String BYE_MESSAGE = "Bye. Hope to see you again soon!";
+    public static final String HORIZONTAL_LINE = "---------------------";
+    public static final String DDL_CMD_SEPARATOR = "/by";
+    public static final String EVENT_CMD_SEPARATOR = "/at";
+    public static final String DONE_CMD = "done";
+    public static final String BYE_CMD = "bye";
+    public static final String TODO_CMD = "todo";
+    public static final String DEADLINE_CMD = "deadline";
+    public static final String EVENT_CMD = "event";
+    public static final String LIST_CMD = "list";
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
-    }
-}
+
+        greet();
+        ArrayList<Task> storedTasks = new ArrayList<>();
+        Scanner in = new Scanner(System.in);
+        String response = in.nextLine();
+
+        while(!response.equals(BYE_CMD)){
+
+            String[] commands = response.split(" ");
+
+            switch (commands[0]) {
+            case DONE_CMD:
+                int taskNumber = parseInteger(commands[1]);
+                if (taskNumber <= storedTasks.size() && taskNumber >= 1){
+                    storedTasks.get(taskNumber - 1).setDone();
+                    displayFormat(storedTasks);
+                } else {
+                    printIllegalCommandWarning();
+                }
+                break;
+            case TODO_CMD:
+                addTodo(storedTasks, commands[1]);
+                break;
+            case DEADLINE_CMD:
+                if(response.contains(DDL_CMD_SEPARATOR)){
+                    String [] arrOfTaskAndTime = extractWords(response);
+                    addDeadline(storedTasks, arrOfTaskAndTime);
+                } else {
+                    printSpecifyTimeWarning();
+                }
+                break;
+            case EVENT_CMD:
+                if(response.contains(EVENT_CMD_SEPARATOR)){
+                    String [] arrOfTaskAndTime = extractWords(response);
+                    addEvent(storedTasks, arrOfTaskAndTime);
+                } else {
+                    printSpecifyTimeWarning();
+                }
+                break;
+            case LIST_CMD:
+                displayFormat(storedTasks);
+                break;
+            default:
+                printIllegalCommandWarning();
+                break;
+            }
+
+            System.out.println("What else?");
+            response = in.nextLine();
+
+        }
+
+        exit();
```

Comment:
A better name for this would be printExit, because exit() sounds like you are terminating the program at this point right away